src/Microsoft.AspNet.SignalR.Owin/Infrastructure/OwinEnvironmentExtensions.cs

```diff
@@ -59,7 +59,7 @@ internal static bool GetIsDebugEnabled(this IDictionary<string, object> environm
             if (environment.TryGetValue(OwinConstants.HostAppModeKey, out value))
             {
                 var stringVal = value as string;
-                return !String.IsNullOrWhiteSpace(stringVal) &&
+                return String.IsNotNullOrWhiteSpace(stringVal) &&
```

Comment:
This isn't a method on `String`. Though we shouldn't make changes to this project.